shared/metrics/src/main/java/io/pravega/shared/MetricsTags.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared;
+
+import com.google.common.base.Preconditions;
+
+public class MetricsTags {
```

Comment:
`public final class`

Reply:
Good catch! Yes this class has nothing to be extended.